packages/backend/src/graphql/resolvers/incoming_payment.test.ts

```diff
@@ -68,4 +102,149 @@ describe('Incoming Payment Resolver', (): void => {
       }
     })
   })
+
+  describe('Mutation.createIncomingPayment', (): void => {
+    test.each`
+      description  | externalRef  | desc
+      ${'rent'}    | ${undefined} | ${'description'}
+      ${undefined} | ${'202201'}  | ${'externalRef'}
+    `('200 ($desc)', async ({ description, externalRef }): Promise<void> => {
+      const { id: paymentPointerId } = await createPaymentPointer(deps, {
+        asset
+      })
+      const payment = await createPayment({
+        paymentPointerId,
+        description,
+        externalRef
+      })
+
+      const createSpy = jest
+        .spyOn(incomingPaymentService, 'create')
+        .mockResolvedValueOnce(payment)
+
+      const input = {
+        paymentPointerId: payment.paymentPointerId,
+        incomingAmount: payment.incomingAmount,
+        expiresAt: payment.expiresAt
+      }
+
+      const query = await appContainer.apolloClient
+        .query({
+          query: gql`
+            mutation CreateIncomingPayment(
+              $input: CreateIncomingPaymentInput!
+            ) {
+              createIncomingPayment(input: $input) {
+                code
+                success
+                payment {
+                  id
+                  state
```

Comment:
Querying all the fields will help test `paymentToGraphql`

Reply:
Should that include all the fields that get returned?
```
const query = await appContainer.apolloClient
        .query({
          query: gql`
            mutation CreateIncomingPayment(
              $input: CreateIncomingPaymentInput!
            ) {
              createIncomingPayment(input: $input) {
                code
                success
                message
                payment {
                  id
                  paymentPointerId
                  state
                  expiresAt
                  incomingAmount {
                    value
                    assetCode
                    assetScale
                  }
                  receivedAmount {
                    value
                    assetCode
                    assetScale
                  }
                  description
                  externalRef
                  createdAt
                }
              }
            }
          `,
          variables: { input }
        })
        .then(
          (query): IncomingPaymentResponse => query.data?.createIncomingPayment
        )

      expect(createSpy).toHaveBeenCalledWith(input)
      expect(query.code).toBe('200')
      expect(query.success).toBe(true)
      expect(query.message).toBeNull()
      expect(query.payment?.id).toBe(payment.id)
      expect(query.payment?.paymentPointerId).toBe(payment.paymentPointerId)
      expect(query.payment?.state).toBe(SchemaPaymentState.Pending)
      expect(query.payment?.expiresAt).toBe(payment.expiresAt.toISOString())
      expect(BigInt(query.payment?.incomingAmount.value)).toBe(payment.incomingAmount.value)
      expect(query.payment?.incomingAmount.assetCode).toBe(payment.incomingAmount.assetCode)
      expect(query.payment?.incomingAmount.assetScale).toBe(payment.incomingAmount.assetScale)
      expect(BigInt(query.payment?.receivedAmount.value)).toBe(payment.receivedAmount.value)
      expect(query.payment?.receivedAmount.assetCode).toBe(payment.receivedAmount.assetCode)
      expect(query.payment?.receivedAmount.assetScale).toBe(payment.receivedAmount.assetScale)
      expect(query.payment.description).toBe(payment.description)
      expect(query.payment.externalRef).toBe(payment.externalRef)
      expect(query.payment.createdAt).toBe(payment.createdAt.toISOString())
```

Reply:
Yeah, and you check `query.payment` in a single expect a la:
https://github.com/interledger/rafiki/blob/6cb56141013d38197b13b31e392ebbee402adcd6/packages/backend/src/graphql/resolvers/outgoing_payment.test.ts#L121-L209

Amount checks from that outgoing payment example could actually be simpified as:
```
            sendAmount: {
              ...serializeAmount(payment.sendAmount),
              __typename: 'Amount'
            },
```
using:
https://github.com/interledger/rafiki/blob/6cb56141013d38197b13b31e392ebbee402adcd6/packages/backend/src/open_payments/amount.ts#L21-L27